app/views/templates/search/search_object.rb

```diff
@@ -299,27 +312,32 @@ def content
             type: render_document_show_field_value(document, 'proxies.dcType')
           },
           refs_rels: data_section( {
-            :title => 'site.object.meta-label.refs-rels',
-            :sections  => [
+            title: 'site.object.meta-label.refs-rels',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.